From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sumit Tiwari <sumit.tiwari@canonical.com>
Date: Sep, 30 2026 10:10:30 +0530
Subject: [PATCH] Add missing po/LINGUAS files to fix FTBFS

Newer gettext macros strictly require a po/LINGUAS file and ignore the 
old ALL_LINGUAS variable. This static patch helps provide the missing 
file to prevent the build from silently generating an empty locale 
directory and failing during dh_install.

Bug-Debian: https://bugs.debian.org/1143643
Bug-Ubuntu: https://bugs.launchpad.net/bugs/2166472
Forwarded: https://bugs.debian.org/1143643
---
--- /dev/null
+++ bluefish-2.4.2/po/LINGUAS
@@ -0,0 +1,34 @@
+ar
+bg
+ca
+cs
+da
+de
+el
+en
+es
+eu
+fa
+fi
+fr
+gl
+hu
+it
+ja
+ko
+nb
+nl
+nn
+pl
+pt
+pt_BR
+ro
+ru
+sk
+sr
+sv
+ta
+tr
+uk
+zh_CN
+zh_TW
